client/styles/components/_toolbar.scss

```diff
@@ -147,3 +154,36 @@
 		accent-color:getThemifyVariable('logo-color');
 	}
 }
+
+.toolbar__duplicatetoedit {
+	display: flex;
+	gap: 2px;
+	cursor: pointer;
+    padding: 3px 8px;
+	align-items: center;
+	justify-content: center;
+    border-radius: 5px;
+
+    &.contrast {
+        background-color: #F5DC23;
+        color: black;
+    }
+
+    &.normal {
+        background-color: $p5js-pink;
+        color: white;
+    }
```

Comment:
See comments re `theme` in components.  We want this to look like `color: getThemifyVariable('secondary-text-color');` (with whatever the correct variable name is).